app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -56,7 +56,24 @@ def show
       end
 
       def update
-        render plain: 'stub to ensure Socure callback exists and the route works'
+        clear_future_steps!
+        idv_session.redo_document_capture = nil # done with this redo
+        # Not used in standard flow, here for data consistency with hybrid flow.
+        document_capture_session.confirm_ocr
+
+        result = handle_stored_result
+        # TODO: new analytics event?
+        analytics.idv_doc_auth_document_capture_submitted(**result.to_h.merge(analytics_arguments))
```

Comment:
```suggestion
```

i see where you're going with this ... it could be helpful to have an event showing the user has in fact returned to IdV 🤔